samza-core/src/test/java/org/apache/samza/zk/TestZkUtils.java

```diff
@@ -265,7 +265,7 @@ public void testRegisterProcessorAndGetIdShouldFailForDuplicateProcessorRegistra
     // Register processor 1 which is not duplicate, this registration should succeed.
     zkUtils.registerProcessorAndGetId(processorData1);
 
-    ZkUtils zkUtils1 = getZkUtils();
+    ZkUtils zkUtils1 = new ZkUtils(KEY_BUILDER, zkClient, 1000, 500, new NoOpMetricsRegistry());
```

Comment:
Can we reduce the value of SESSION_TIMEOUT_MS and CONNECTION_TIMEOUT_MS constants that getZkUtils uses?

Reply:
Makes sense. Done.